integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java

```diff
@@ -87,6 +87,11 @@ public ContainerAllocator(String containerName, int targetNumContainers, int max
     mAllocatedContainers = new ArrayList<>();
   }
 
+  /**
+   * Gets the hosts of workers that could be allocated.
+   *
+   * @return the unused worker hosts
```

Comment:
"@return a list of all hosts in the cluster which haven't yet reached the containers per host limit."